tools/styles/package.json

```diff
@@ -112,10 +113,10 @@
         "@spectrum-web-components/base": "^0.43.0"
     },
     "devDependencies": {
-        "@spectrum-css/commons": "^10.0.0",
+        "@spectrum-css/commons": "^11.0.0-s2-foundations.9",
         "@spectrum-css/expressvars": "^3.0.9",
-        "@spectrum-css/tokens": "^14.0.0",
-        "@spectrum-css/tokens-v2": "npm:@spectrum-css/tokens@^14.0.0-next.3",
+        "@spectrum-css/tokens": "^@spectrum-css/tokens@14.3.1",
+        "@spectrum-css/tokens-v2": "npm:@spectrum-css/tokens@^15.0.0-s2-foundations.9",
         "@spectrum-css/typography": "^6.1.0",
```

Comment:
@TarunAdobe should this be `"@spectrum-css/typography": "^7.0.0-s2-foundations.9",`

Reply:
This was creating some errors back when I tried... Will revisit this again tommoorrow. Thank you for reminding